### PR TITLE
[Showcase] Use streamlit-webrtc python package and computer webcam to live streaming image detection when the yolov4-tiny model is deployed on PrimeHub deployment

### PR DESCRIPTION
**Pull request by LiuYuWei**
Why:
We can demo PrimeHub deployment Image detection in PrimeHub App.
How:
Upload the related folder.

### CODE REVIEW

**Review by jimmyliao — APPROVED** [review]
LGTM. Please help to resolve the conflict before merge.

**Comment by LiuYuWei** [comment]
@jimmyliao Done. Please merge this after #15 is merged.

**Pull request merged**
